Index: file-browser.js
===================================================================
--- file-browser.js	before
+++ file-browser.js	after
@@ -1,7 +1,4 @@
     this.clearButton.addEventListener('click', () => {
-      this.fileSearchInput.value = '';
-      this.fileSearchQuery = '';
-      this.filterContents();
-      this.updateClearButton();
+      this.clearSearchInput();
       this.fileSearchInput.focus();
     });
\ No newline at end of file
